### ISSUE TITLE
Fix contrast issues on truss.works

### ISSUE THREAD

**Issue by xtine** [trussworks/accessibility#1]
Issue 2: Contrast between hero and white text does not pass WCAG
Contrast between description text over hero image does not pass WCAG AA. WCAG states that the minimum contrast ratio for small text (18pt/24px and below) is 4.5:1.

Issue 3: Contrast on buttons does not meet WCAG
Contrast between description text over hero image does not pass WCAG AA. WCAG states that the minimum contrast ratio for small text (18pt/24px and below) is 4.5:1.

Issue 4: Text links do not meet WCAG for contrast
Text links with Truss orange do not pass WCAG AA for contrast. WCAG states that the minimum contrast ratio for small text (18pt/24px and below) is 4.5:1.

Doc with all documented accessibility issues:
https://docs.google.com/document/d/1upM9ue4UAuJwIBbZ6fRdjGKmzs-odcWxDQfYftIrHyw/edit?usp=sharing

PT IMPORTED STORY: https://www.pivotaltracker.com/story/show/171138328

**Comment by xtine** [comment]
Fixed index page contrast issues:
- Darkened orange link color to #C94A31
- Changed alternate section background color to: #FBFCFE

**Comment by xtine** [comment]
Fixed button contrast issues by darkening orange colors to: #C94A31

**Comment by xtine** [comment]
Darkened header color to: #C94A31

**Comment by xtine** [comment]
The only issue remaining is number 2 on the Google doc, which is the contrast issue between hero image and the white text above it. I've tagged @joshmfranklin to look into how we should approach this issue.
